build/DependencyVersions.props

```diff
@@ -58,15 +58,12 @@
   <PropertyGroup>
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25908-02</CLI_NETStandardLibraryNETFrameworkVersion>
 
+    <!-- When ready to pull aspnetcore directly from orchestrated build, move this above the import to OrchestratedPackageVersionsProps. -->
+    <MicrosoftAspNetCoreAppPackageVersion>2.1.0-preview1-28031</MicrosoftAspNetCoreAppPackageVersion>
+
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
-    <!-- This is the version for ASP.NET packages that the CLI depends on to build, like the
-         component for generating HTTPS certificates on first run. -->
-    <CLI_ASPNET_Version>2.1.0-preview1-27617</CLI_ASPNET_Version>
-    <AspNetCoreTemplatePackageVersion>2.0.3</AspNetCoreTemplatePackageVersion>
-    <AspNetCoreRuntimePackageBrandName>aspnetcore-store</AspNetCoreRuntimePackageBrandName>
-    <AspNetCoreRuntimePackageFolderName>dev-26623</AspNetCoreRuntimePackageFolderName>
-    <AspNetCoreRuntimePackageVersion>2.1.0-preview1-26623</AspNetCoreRuntimePackageVersion>
+    <AspNetCoreVersion>$(MicrosoftAspNetCoreAppPackageVersion)</AspNetCoreVersion>
```

Comment:
Why are we redefining this? Why not just use "MicrosoftAspNetCoreAppPackageVersion"?

Reply:
Was trying to be consistent with how package versions are used to infer versions of other things. e.g.
```
<SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
<HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
<AspNetCoreVersion>$(MicrosoftAspNetCoreAppPackageVersion)</AspNetCoreVersion>
```